app/Dao/Doctor/DoctorDao.php

```diff
@@ -3,11 +3,29 @@
 namespace App\Dao\Doctor;
 
 use App\Contracts\Dao\Doctor\DoctorDaoInterface;
-use Illuminate\Http\Request;
+use App\Models\Doctor;
 
 /**
  * Data accessing object for doctor
  */
 class DoctorDao implements DoctorDaoInterface
 {
+     /**
+     * To get doctor list
+     * @return $doctors
+     */
+    public function index()
+    {
+        $doctors =  Doctor::paginate(5);
+        return $doctors;
+    }
+     /**
+     * To show doctor detail by id
+     * @param string $id doctor id
+     */
+    public function show($id)
+    {
+        $doctor = Doctor::where('id', $id)->first();
```

Comment:
If param-id not match in doctor's id, show 404 page.
`firstOrFail()` instead of `first()`.